test/format/t.c

```diff
@@ -175,6 +189,9 @@ main(int argc, char *argv[])
         case 'q': /* Quiet */
             quiet_flag = true;
             break;
+        case 'R': /* Reopen */
+            g.reopen = true;
```

Comment:
While I'm okay with this choice, every time I see `-R` I think of it as forcing recovery which is what `-R` does in the `wt` command. I don't have a better suggestion, but just thought I'd say it in case something comes to mind for you.

Reply:
@sue and I discussed offline, and didn't come up with anything better.